src/main/java/dev/andrylat/vomelianchuk/CardNumberOperations/ConsoleUtils.java

```diff
@@ -0,0 +1,15 @@
+package dev.andrylat.vomelianchuk.CardNumberOperations;
+
+import java.util.Scanner;
+
+public class ConsoleUtils {
```

Comment:
Imagine situation that your project lives during last 10 years and this class contains 1000 different methods related with "Console", because it is "utils".

You should always try to make specific meaning for class and don't give possibility to another guys to put here some another methods. 

You can use name like "UserConsoleIO" and it will be enough. Utils here is not appropriate. Utils we use only for classes which are very common. In the best application these classes should be absent./

Reply:
Fixed here: 1783555